src/ILCompiler.ReadyToRun/src/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -825,11 +825,14 @@ private static bool IsDynamicStatics(TypeDesc type)
             MemoryHelper.FillMemory((byte*)pResult, 0xcc, Marshal.SizeOf<CORINFO_CALL_INFO>());
 #endif
             pResult->codePointerOrStubLookup.lookupKind.needsRuntimeLookup = false;
-            useInstantiatingStub = false;
 
             originalMethod = HandleToObject(pResolvedToken.hMethod);
             TypeDesc type = HandleToObject(pResolvedToken.hClass);
 
+            // This formula roughly corresponds to CoreCLR CEEInfo::resolveToken when calling GetMethodDescFromMethodSpec
+            // (that always winds up by calling FindOrCreateAssociatedMethodDesc) around line 1140 in vm/jitinterface.cpp.
```

Comment:
This line number will become outdated fast. If you need to link this to the specific line in CoreCLR, it is better to use github URL with sha1 hash.

Reply:
Thanks for reminding me of due diligence w.r.t. permanent links, I have fixed both occurrences in 2nd commit.